shell/components/templates/home.vue

```diff
@@ -58,6 +62,7 @@ export default {
     <AwsComplianceBanner />
     <AzureWarning />
     <PromptModal />
+    <ModalManager />
```

Comment:
I don't necessarily think we want to change things given this is being accessed via the API but I'm reminded of [teleport](https://vuejs.org/guide/built-ins/teleport).

Reply:
I never knew about teleport but that's a good idea.
I noticed we already had a div with `id="modals"` in our `public/index.html`. So I added the Teleport for the ModalManager to point to `#modals`, and I also created a new div in our index.html for `id="slides"` that we can use for the SlideInPanelManager component.